docs/pages/operations-guide.md

```diff
@@ -355,6 +355,10 @@ hostname:LIKE:10.0.0.159|10.0.1.202|10.0.3.3
 
 You must include spare capacity in this list, so that if one of the whitelisted systems goes down, there is still enough room to repair your service (via [`pod replace`](#replace-a-pod)) without requiring that system.
 
+### Zones
```

Comment:
I am intentionally not documenting Regions in the operations guide as the SDK is not yet region aware.